lib/sass_spec/directory.rb

```diff
@@ -88,21 +104,38 @@ def glob(pattern)
 
   # Returns whether a file exists at `path` within this directory.
   def file?(path)
-    return @archive[path].is_a?(HRX::File) if hrx?
-    File.exist?(File.join(@path, path))
+    if hrx?
+      @archive[path].is_a?(HRX::File)
+    elsif (dir, basename = split_if_nested(path))
```

Comment:
(Here and below) looks like these assignments might not be valid?

Reply:
Updated the Ruby version to avoid this issue.